connect/proxy/proxy_test.go

```diff
@@ -82,3 +83,7 @@ func TestProxy_public(t *testing.T) {
 func testLogger(t *testing.T) *log.Logger {
 	return testutil.LogShim(testutil.Logger(t))
 }
+
+func testLogger2(t *testing.T) hclog.Logger {
```

Comment:
Did you intend to use this?

Reply:
Hmm, nope not sure when that was added. Good catch, will delete.

Reply:
Noticed that the `testLogger` above is also unused, will delete that as well.